docs/intro/install.rst

```diff
@@ -89,15 +89,14 @@ Windows
      Python 3 is not supported on Windows. This is because Scrapy core requirement Twisted does not support
      Python 3 on Windows.
 
-Ubuntu 9.10 or above
---------------------
+Ubuntu 12.04 or above
+---------------------
 
 **Don't** use the ``python-scrapy`` package provided by Ubuntu, they are
-typically too old and slow to catch up with latest Scrapy.
+typically too old and slow to catch up with latest Scrapy. You will need
+to install the following system packages::
 
-Instead, use the official :ref:`Ubuntu Packages <topics-ubuntu>`, which already
-solve all dependencies for you and are continuously updated with the latest bug
-fixes.
+    sudo apt-get install python-pip python-twisted python-openssl python-lxml
```

Comment:
hm, right, I thought "You can install Scrapy with pip after that" is only for the second set of packages. 

I'd drop `python-pip python-twisted python-openssl python-lxml` instructions though because even though they have a slight advantage of installing faster, package versions can be outdated, it won't work in Python 3, and it won't work in virtualenv. 

by the way, `pip install Scrapy` won't work with either set on system dependencies - users need to either create a virtualenv (a great idea), use `sudo pip install Scrapy` (a bad idea), or use `pip install --user Scrapy` (a good idea, but requires some effort to get `scrapy` command-line tool working).